css/style.css

```
@@ -0,0 +1,230 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
+body {
+  line-height: 1;
+  overflow-x: hidden;
+}
+
+.promo {
+  max-width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.header {
+  position: absolute;
+  display: flex;
+  width: 160rem;
+  height: 12.2rem;
+  justify-content: center;
+  align-items: center;
+  padding: 0 16.8rem;
+  background: linear-gradient(to bottom, black -100%, transparent 90%);
+  gap: 0.8rem;
+}
+
+.logo-menu-bar {
+  height: inherit;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: inherit;
```

I bet it's redundant
```suggestion
  display: flex;
  justify-content: space-between;
  align-items: center;
```

---

fixed